iothub/device/src/InternalClient.cs

```diff
@@ -1915,9 +1931,9 @@ internal Task UpdatePropertiesAsync(ClientPropertyCollection propertyPatch, Canc
             }
         }
 
-        internal Task SubscribeToWritablePropertyEventAsync(Func<ClientPropertyCollection, object, Task> callback, object userContext, PayloadConvention payloadConvention, CancellationToken cancellationToken)
+        internal Task SubscribeToWritablePropertyEventAsync(Func<ClientPropertyCollection, object, Task> callback, object userContext, CancellationToken cancellationToken)
```

Comment:
I know that this is not the focus of this PR, but should we rename this as well - SubscribeToWritablePropertiesEventAsync?